docs/detections/detections-ui-exceptions.asciidoc

```diff
@@ -160,7 +160,7 @@ Closes all alerts that match the exception's conditions and were generated only
 [[endpoint-rule-exceptions]]
 === Add Elastic {endpoint-sec} exceptions
 
-Like detection rule exceptions, you can add Endpoint agent exceptions either by editing Elastic {endpoint-sec} rules or by adding them as an action on alerts generated by {endpoint-sec} rules. Elastic {endpoint-sec} alerts have the following fields:
+Like detection rule exceptions, you can add Endpoint agent exceptions either by editing the Elastic {endpoint-sec} rule or by adding them as an action on alerts generated by the {endpoint-sec} rule. Elastic {endpoint-sec} alerts have the following fields:
```

Comment:
I'm not clear on this note. It seems like at least the last sentence should have `{endpoint-cloud-sec}` instead of `{endpoint-sec}`, and it's confusing that the rule called "Endpoint Security" (if I understand correctly) generates detection alerts whenever alerts are received from Elastic Endpoint _and Cloud_ Security. The About section of the "Endpoint Security" rule also refers to "Elastic Endpoint Security" - but if that's now called "Endpoint and Cloud Security", I think the About should be updated, as well as the rule's name. Do you agree? Should we file a bug?

Reply:
@benironside Good catch, and you're right — it's confusing that the Endpoint Security rule hasn't been renamed yet, so we kind of have to write around that. But we shouldn't keep using the `{endpoint-sec}` attribute; we should hard-write "Endpoint Security" when referring to the rule. I'll revisit this section and push a new commit soon.

I [asked about](https://github.com/elastic/security-team/issues/3928#issuecomment-1151127762) renaming the rule, and it'll be worth re-pinging the folks who'd decide that, but at least for 8.3.0 it looks like the rule name won't change. If/when it does we can revisit these direct references to the rule.